doc/reference/reference_rock/vshard/vshard_router.rst

```diff
@@ -768,6 +770,46 @@ Router public API
             status: available_rw
         ...
 
+..  _router_api-enable:
+
+..  function:: vshard.router.enable()
+
+    Manually allow access to the router API, revert
+    :ref:`vshard.router.disable() <router_api-disable>`.
+
+        ..  note::
+
+            ``vshard.router.enable()`` cannot be used for enabling a router
+            API that was automatically disabled due to a running configuration
```

Comment:
Actually, `vshard.router.cfg` is a configuration process only for the static router. But vshard supports multiple routers in a single instance. These ones can be created via `vhard.router.new`. During configuration every router is blocked on its own,  and blocking API of the one uncofigured instance doesn't affect others. 

If we want to specify functions, used for configuration, explicitly, we should mention both of them. Another solution can be just stay on 'running configuration process'.